### PR TITLE
(WIP) Check pickling of ufuncs - reopened

### PR DESCRIPTION
**Pull request by sdegryze**
reopened in an attempt to fix triggering travis build

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,9 @@
 test_requirements = [
     'pytest',
     'pytest-cov',
-    'mock'
+    'mock',
+    'numpy',
+    'scipy'
 ]
```
Comment (ogrisel) [inline setup.py]: The `test_requirements` have been dropped from the master branch in favor of a `dev-requirements.txt` file. As installing numpy / scipy is currently very costly under Linux (built from source when installed with pip or setuptools) I would rather not put them as a dependency in `dev-requirements.txt`) but instead upgrade the travis config to use miniconda, at least on some of the entries of the build matrix and only put those dependencies in the `.travis.yml` file.

**Comment by ogrisel** [comment]
@sdegryze if you want I can help do the squashing & rebasing of this PR.

**Comment by sdegryze** [comment]
Thanks @ogrisel - it took me a while but it looks like Travis is happy now. I have no clue why builds were not triggered before. I rebased, squashed the commits, and opened a new PR (https://github.com/cloudpipe/cloudpickle/pull/34). Still learning how to go through these steps - appreciate your patience :)

**Comment by sdegryze** [comment]
Closing this PR since a squashed version is opened under #34

**Comment by ogrisel** [comment]
Note for later: you can `git push --force` the rebased / squashed version to the same branch to get the same pull request updated. No need to create a new one.

**Pull request closed without merging**